app/components/buttons/WatchOnlyWalletSwitch.js

```diff
@@ -0,0 +1,25 @@
+import EventListener from "react-event-listener";
+import { Tooltip } from "shared";
+import { InvisibleButton } from "buttons";
+import { FormattedMessage as T } from "react-intl";
+import "style/EyeFilterMenu.less";
+import "style/StakePool.less";
+
+const WatchOnlyWalletSwitch = ({ enabled, onClick, className }) => (
+  <div className={className ? className : ""}>
+    <Tooltip text={enabled ? <T id="autobuyer.enabled" m="Watch only Wallet" /> : <T id="autobuyer.disabled" m="Not Watch only Wallet" />}>
+      <EventListener target="document">
+        <div className="teste">
```

Comment:
Why do you have this EventListener here? Also, change `className="teste"` to something more meaningful or remove altogether if not needed